script/index.js

```diff
@@ -1,16 +1,40 @@
-const hamburger = document.querySelector('.mobile-menu');
-const closeHamburger = document.querySelector('.desktop-nav-hidden');
-const nav = document.querySelector('.desktop-nav-hidden');
-const closeNav = document.querySelector('.desktop-nav-hidden');
+// Toggle Nav Menu
 
-hamburger.addEventListener('click', () => {
-  nav.classList.add('openMenu');
-});
+const menu = document.querySelector('.nav__menu');
+const menuBtn = document.querySelector('#open-menu-btn');
+const closeBtn = document.querySelector('#close-menu-btn');
+const cardBtn = document.querySelectorAll('.card-button');
+const navLink = document.querySelector('#nav-menu');
+const popBtn = document.querySelector('.close-btn');
+const popUp = document.querySelector('#popup-1');
 
-closeHamburger.addEventListener('click', () => {
-  nav.classList.remove('openMenu');
+menuBtn.addEventListener('click', () => {
+  menu.style.display = 'flex';
+  closeBtn.style.display = 'inline-block';
+  menuBtn.style.display = 'none';
+  menu.style.zIndex = '10';
+  closeBtn.style.zIndex = '11';
 });
 
-closeNav.addEventListener('click', () => {
-  nav.classList.remove('openMenu');
+// Close the nav menu
+
+const closeNav = () => {
+  menu.style.display = 'none';
+  closeBtn.style.display = 'none';
+  menuBtn.style.display = 'inline-block';
+};
+// close nav menu on clicking close button
+closeBtn.addEventListener('click', closeNav);
+
+// Close nav menu on clicking any link item
+navLink.addEventListener('click', closeNav);
+
+cardBtn.forEach((item) => {
+  item.addEventListener('click', () => {
+    document.querySelector('#popup-1').classList.toggle('active');
+  });
 });
+
+popBtn.addEventListener('click', () => {
+  popUp.style.display = 'none';
+});
```

Comment:
I can only open the modal window once after page reload. Every subsequent attempt at opening a mobile window throws an error in the console. The reason for that is that you are applying an `onclick` attribute directly to the HTML calling a `popup()` method that becomes null after the first click. You should have an array or an object that holds the section information and create a function that opens and closes the modal window for each project.

<img width="1109" alt="Screen Shot 2022-06-24 at 8 29 00" src="https://user-images.githubusercontent.com/82426682/175545913-85f73188-eadc-4c78-87f6-30f4d390a61b.png">

Please fix this.